app/views/categories/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p style="color: green"><%= notice %></p>
+
+<%= render @category %>
+
+<div>
+  <%= link_to "Edit this category", edit_category_path(@category) %> |
+  <%= link_to "Back to categories", categories_path %>
```

Comment:
Nice job so far👍🏼. You have a great UI implementation😉. But just so we can stick to the design requirements of this milestone, let's take away the `back to categories` button😉. and make the  `add transaction` button match the design😉. do this for all pages where necessary👍🏼.

design
<img width="294" alt="image" src="https://github.com/Rifat8080/budget-app/assets/64643995/2e897c91-8419-44fd-8176-2a8b0bfa7d32">


yours
<img width="377" alt="image" src="https://github.com/Rifat8080/budget-app/assets/64643995/ac5fb2fd-e65e-405c-a1d2-714b96d90a40">


For the button that navigates the user to the previous page, just as [required](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md), we can make use of the `<` icon button to replace the hamburger icon😉. and give it the functionality of navigating to the previous page😉.

<img width="992" alt="Screenshot 2024-01-11 at 09 46 23" src="https://github.com/Rifat8080/budget-app/assets/64643995/abaa9d74-41ce-45bd-8dfb-17b1e47bb1c1">


something like what you have below, with the green highlighter. 
<img width="384" alt="Screenshot 2024-01-11 at 09 48 18" src="https://github.com/Rifat8080/budget-app/assets/64643995/bdb359ef-22ef-4f13-9df4-00069618749c">


**If you require more clarity on this, you can send a message on slack or call me on zoom😉**